src/screens/NoteEditScreen.tsx

```diff
@@ -72,15 +74,11 @@ export default function NoteEditScreen(props: PropsType) {
       setContent_(content);
       setLoading(false);
     })();
-
-    return () => {
-      onSaveDoRef.current?.();
-    };
-  }, []);
+  }, [syncGate, colUid, itemUid]);
```

Comment:
Just one tiny nitpick: this should probably be syncGate, colUid, itemUid and cacheItem. First of all for linters that check for this, but also in case the item changes for whatever reason due to a refresh.

Reply:
The thing is, this is only supposed to load the content when the screen loads. After that setContent is handled by the editor.